tests/lib/external/snapd-testing-tools/utils/log-parser

```diff
@@ -591,7 +575,13 @@ for the error/debug/warning output.
         "--print-results",
         type=str,
         default=NONE,
-        choices=[ALL, FAILED, ABORTED, SUCCESSFUL, NONE],
+        choices=[
+            ALL,
+            helper.FAILED.lower(),
```

Comment:
This is not very nice, can we replace this with an enum that is not reusing other vales from other places?